README.md

```diff
@@ -12,15 +14,37 @@ The input to the pipeline is a standard sample sheet (passed as `--input samples
 | ------- | ----------------- | ---------- | ---------- | ---------- | ---------- | ---------- | ---------- | ---------- | ---------- |
 | SampleA | sampleA.mlst.json | meta1      | meta2      | meta3      | meta4      | meta5      | meta6      | meta7      | meta8      |
 
-The structure of this file is defined in [assets/schema_input.json](assets/schema_input.json). Validation of the sample sheet is performed by [nf-validation](https://nextflow-io.github.io/nf-validation/).
+The structure of this file is defined in [assets/schema_input.json](assets/schema_input.json). Validation of the sample sheet is performed by [nf-validation](https://nextflow-io.github.io/nf-validation/). Details on the columns can be found in the [Full samplesheet](docs/usage.md#full-samplesheet) documentation.
 
 # Parameters
 
 The main parameters are `--input` as defined above and `--output` for specifying the output results directory. You may wish to provide `-profile singularity` to specify the use of singularity containers and `-r [branch]` to specify which GitHub branch you would like to run.
 
 ## Metadata
 
-In order to customize metadata headers, the parameters `--metadata_1_header` through `--metadata_8_header` may be specified.
+In order to customize metadata headers, the parameters `--metadata_1_header` through `--metadata_8_header` may be specified. These parameters are used to re-name the headers in the final metadata table from the defaults (e.g., rename `metadata_1` to `country`).
```

Comment:
Just to confirm - this means that the input samplesheet HAS to have headers such as `meta1` (or `metadata_1`?) and changing the renaming HAS to be done by the `--metadata_1_header` parameter in the command-line. A user cannot upload a sample sheet that presets custom headers like:

| --------- | -------------------- | --------- |
| SampleA | sampleA.mlst.json  | country |

Reply:
Yes, this is exactly correct. The samplesheet has to have the headers `metadata_1`, ..., `metadata_8`, with renaming done by passing new header names as parameters `--metadata_1_header`.

It is not ideal and there may be other solutions. The issue we are trying to address is how to pass metadata via IRIDA Next. We want to pass metadata in the samplesheet.csv, which uses `schema_input.json` to define the column names. This means column names in the samplesheet must be fixed and so passing new names via parameters was the solution we came up with.

We may be able to revisit this later to find a better solution for passing metadata.

Reply:
Thank you for clarifying and makes sense for the `schema_input.json` to require fixed names.